src/components/daily/physicaltooltip/PhysicalTooltip.js

```diff
@@ -101,7 +101,7 @@ PhysicalTooltip.propTypes = {
     left: PropTypes.number,
     horizontal: PropTypes.number,
   }),
-  titls: PropTypes.node,
+  title: PropTypes.node,
```

Comment:
Missing `isRequired` or a default value.

Reply:
it's an optional property. 
Why would I need to set a default value ?

Reply:
That's how PropType works and have been think.
You can set the default value to null for example.